rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -539,7 +539,8 @@ void Compiler::VCMPGTUW_(u32 vd, u32 va, u32 vb) {
 void Compiler::VCTSXS(u32 vd, u32 uimm5, u32 vb) {
 	auto vb_v4f32 = GetVrAsFloatVec(vb);
 	if (uimm5) {
-		vb_v4f32 = m_ir_builder->CreateFMul(vb_v4f32, m_ir_builder->CreateVectorSplat(4, ConstantFP::get(m_ir_builder->getFloatTy(), 1 << uimm5)));
+		u64 float_value = UINT64_C(1) << uimm5;
+		vb_v4f32 = m_ir_builder->CreateFMul(vb_v4f32, m_ir_builder->CreateVectorSplat(4, ConstantFP::get(m_ir_builder->getFloatTy(), reinterpret_cast<double&>(float_value))));
```

Comment:
https://www.power.org/wp-content/uploads/2013/05/PowerISA_V2.07_PUBLIC.pdf
```
For each integer value i from 0 to 3, do the following.
Single-precision floating-point word element i in
VRB is multiplied by 2UIM. The product is
converted to a 32-bit signed fixed-point integer
using the rounding mode Round toward Zero.
```
So it was correct. If static analyzer doesn't want implicit casts, use `static_cast<double>`.

This is an example of when blindly trying to silence warnings may lead to quiet errors.